src/main/play-doc/operation/Install.md

```diff
@@ -1235,7 +1245,7 @@ frontend conductr-haproxy-test
   mode http
   monitor-uri /test
 
-{{#eachAcls bundles defaultHttpPort=9000}}
+{{#eachAcls bundles defaultHttpPort=80}}
```

Comment:
I think our default template exposes HTTP based service via the proxy on port 9000. Why do we change it to port 80?